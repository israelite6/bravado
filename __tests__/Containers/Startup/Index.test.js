/**
 * @format
 */

import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import IndexStartupContainer from '../../../src/Containers/Startup/index';

it('Startup component renders correctly', () => {
  const tree = renderer.create(<IndexStartupContainer />).toJSON();
  expect(tree).toMatchSnapshot();
});
