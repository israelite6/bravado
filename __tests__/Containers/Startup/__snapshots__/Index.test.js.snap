// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Startup component renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "paddingHorizontal": 15,
      "paddingVertical": 5,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "borderRadius": 4,
        "flexDirection": "row",
        "paddingHorizontal": 12,
        "paddingVertical": 10,
      }
    }
  >
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "color": undefined,
            "fontSize": 10,
          },
          Object {
            "color": "#B3B3B3",
            "paddingRight": 9,
          },
          Object {
            "fontFamily": "anticon",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      î™°
    </Text>
    <TextInput
      allowFontScaling={true}
      defaultValue=""
      onChangeText={[Function]}
      rejectResponderTermination={true}
      style={
        Object {
          "flexGrow": 1,
          "fontSize": 14,
          "paddingVertical": 0,
        }
      }
      underlineColorAndroid="transparent"
    />
  </View>
  <RCTScrollView
    data={Array []}
    disableVirtualization={false}
    getItem={[Function]}
    getItemCount={[Function]}
    horizontal={false}
    initialNumToRender={10}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    numColumns={1}
    onContentSizeChange={[Function]}
    onEndReachedThreshold={2}
    onLayout={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={Array []}
    style={
      Object {
        "flex": 1,
        "marginVertical": 5,
      }
    }
    updateCellsBatchingPeriod={50}
    viewabilityConfigCallbackPairs={Array []}
    windowSize={21}
  >
    <View />
  </RCTScrollView>
</View>
`;
